.github/workflows/verify.yaml

```diff
@@ -1,11 +1,6 @@
 name: 'VIB'
```

Comment:
Why did you change the name? We should follow the same approach as [here](https://github.com/bitnami/charts/tree/master/.github/workflows):

- `.github/workflows/vib-verify.yaml`: GitHub Pipeline to integrate with Content Platform and perform a verification.
- `.github/workflows/vib-publish.yaml`: GitHub Pipeline to integrate with Content Platform and perform a publishing.

Reply:
There was no reason to change it. Good catch